rest-tests/src/test/java/io/syndesis/qe/rest/tests/publicapi/EnvironmentsPublicApiSteps.java

```diff
@@ -0,0 +1,50 @@
+package io.syndesis.qe.rest.tests.publicapi;
+
+import cucumber.api.java.en.When;
+import io.syndesis.qe.endpoints.publicendpoint.EnvironmentsPublicEndpoint;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+public class EnvironmentsPublicApiSteps {
+
+    @Autowired
+    private EnvironmentsPublicEndpoint environmentsEndpoint;
+
+    @When("check that number of tags is (\\w+)$")
+    public void checkNumberOfTags(int numberOfTags) throws IOException {
+        List<String> list = getAllTags();
+        assertThat(list.size()).isEqualTo(numberOfTags);
```

Comment:
I'd say that the `list` variable and `getAllTags()` method can be left out and 

```
assertThat(environmentsEndpoint.getAllEnvironments().size()).isEqualTo(numberOfTags)
```

should be enough. wdyt?

Reply:
and all methods that are checking something should be `@Then`